be/src/exec/orc_builder.cpp

```diff
@@ -0,0 +1,462 @@
+// This file_builder is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#include "orc_builder.h"
```

Comment:
Consider to put this file into `src/formats/orc`, and rename it like `orc_chunk_builder`.

Reply:
And could you add some test cases for this class?